src/main/java/com/easypost/model/Shipment.java

```diff
@@ -941,41 +920,6 @@ public Shipment buy(final Map<String, Object> params) throws EasyPostException {
         return this.buy(params, false, null);
     }
 
-    /**
-     * Buy this Shipment.
-     *
-     * @param params           the options for the query.
-     * @param withCarbonOffset whether to include a carbon offset when buying the shipment.
-     * @return Shipment object
-     * @throws EasyPostException when the request fails.
-     */
-    public Shipment buy(final Map<String, Object> params, final boolean withCarbonOffset) throws EasyPostException {
-        return this.buy(params, withCarbonOffset, null);
-    }
```

Comment:
Issue: This function must remain because users could pass the rate via the `params` parameter.